src/View/View.php

```diff
@@ -15,78 +18,110 @@
+     *             extensions.
+     */
+    public $defaultExtension = 'php';
+
     /**
      * @var mixed custom parameters that are shared among view templates.
      */
     public $params = [];
+
+    /**
+     * @var EventDispatcherInterface $eventDispatcher
+     */
+    protected $eventDispatcher;
+
     /**
-     * @var array a list of available renderers indexed by their corresponding supported file extensions.
-     *            Each renderer may be a view renderer object or the configuration for creating the renderer object.
-     *            For example, the following configuration enables the Twig view renderer:
+     * @var array a list of available renderers indexed by their corresponding supported file extensions. Each renderer
+     *            may be a view renderer object or the configuration for creating the renderer object. For example, the
+     *            following configuration enables the Twig view renderer:
      *
      * ```php
      * [
      *     'twig' => ['__class' => \Yiisoft\Yii\Twig\ViewRenderer::class],
      * ]
      * ```
      *
-     * If no renderer is available for the given view file, the view file will be treated as a normal PHP
-     * and rendered via PhpTemplateRenderer.
+     * If no renderer is available for the given view file, the view file will be treated as a normal PHP and rendered
+     * via PhpTemplateRenderer.
      */
-    protected $renderers = [
-    ];
-    /**
-     * @var string the default view file extension. This will be appended to view file names if they don't have file extensions.
-     */
-    public $defaultExtension = 'php';
+    protected $renderers = [];
+
     /**
      * @var Theme the theme object.
      */
     protected $theme;
-    /**
-     * @var array a list of named output blocks. The keys are the block names and the values
-     *            are the corresponding block content. You can call [[beginBlock()]] and [[endBlock()]]
-     *            to capture small fragments of a view. They can be later accessed somewhere else
-     *            through this property.
-     */
-    public $blocks;
+
     /**
      * @var DynamicContentAwareInterface[] a list of currently active dynamic content class instances.
      */
-    private $_cacheStack = [];
-    /**
-     * @var Locale source locale used to find localized view file.
-     */
-    private $_sourceLocale;
+    private $cacheStack = [];
+
     /**
      * @var array a list of placeholders for embedding dynamic contents.
      */
-    private $_dynamicPlaceholders = [];
+    private $dynamicPlaceholders = [];
+
     /**
-     * @var array the view files currently being rendered. There may be multiple view files being
-     *            rendered at a moment because one view may be rendered within another.
+     * @var string $language
      */
-    private $_viewFiles = [];
-
-    protected $eventDispatcher;
-
-    private $basePath;
+    private $language = 'en';
```

Comment:
It is very hard to review since properties are reordered :(

Reply:
Sorry :(